test/cp.js

```diff
@@ -451,6 +451,45 @@ test('-R implies -P', t => {
   });
 });
 
+test('-Ru respects the -u flag recursively (don\'t update newer file)', t => {
```

Comment:
You **might** be able to replace this with a template literal to avoid escaping the apostrophe. I can't remember if our lint currently forbids this (see #788). If lint complains, just leave as-is.